h/services/search_index/_queue.py

```diff
@@ -180,32 +180,37 @@ def sync(self, limit):
 
         LOG.info(dict(counts))
 
-    def count(self, tags=None):
-        query = self._job_query
-
-        if tags:
-            query = query.filter(Job.tag.in_(tags))
-
-        return query.count()
+    def count(self, tags=None, expired=False):
+        return self._job_query(tags=tags, expired=expired).count()
```

Comment:
:+1: I think this is more consistent with the way the code is going. One method to rule them all, for making queries at least.